doc/Release_Build_Creation.rst

```diff
@@ -1,16 +1,30 @@
-=======================
-Release Build Creation
-=======================
+==================================
+Cortx Stack Release Build Creation
+==================================
 
 This file consists of the procedure that should be followed to generate the release build outside the Seagate network using `cortx-build <https://github.com/orgs/Seagate/packages/container/package/cortx-build>`_ docker image. 
 
+*****************************************
+Specifications to setup Virtual Machine
+*****************************************
+
+- Single-Node VM deployment
```

Comment:
I thought we are planning to add separate file for specifications and add links to it. Also specifications are for Deployment and not for compilation.

Reply:
yes, but thought to include in one file. Also included in compilation as this will be the first step for the community to setup VM & compile the build

Reply:
Ok. I guess we already discussed it at - https://github.com/Seagate/cortx/pull/1049#discussion_r634215858 
May be we can take opinion of other reviewers and if looks good to then go ahead.

Reply:
I think we should do this, as @christina-00 said if you do not have these disks and NICs you will have to restart the entire process again and we do not want to let anyone have to do this.

Reply:
done